### ISSUE TITLE
gruvbox theme linefly inconsistent colors

### ISSUE THREAD

**Issue by qbnil** [bluz71/nvim-linefly#13]
![image](https://github.com/bluz71/nvim-linefly/assets/130576530/46da92b6-0259-486d-9278-c55004264e35)
the blue color appeared out of nowhere..

**Comment by bluz71** [comment]
Which exact gruvbox theme? Please provide link to the gruvbox you are using thanks.

**Comment by qbnil** [comment]
https://github.com/ellisonleao/gruvbox.nvim - the original one

**Comment by bluz71** [comment]
Hmmm, it appears to work for me:

![gruvbox](https://github.com/bluz71/nvim-linefly/assets/11382509/7172c2f9-9f05-44fa-aab6-c06cbc5a0861)

**Comment by bluz71** [comment]
The above screenshot is GNOME Terminal.
Alacritty also works for me.

Can you experiment with a different font AND a different terminal.

Are you using Neovim 0.10? That's what I am using.

**Comment by qbnil** [comment]
yep, just making it clear: i use kitty on arch with Neovim v0.10.0

**Comment by bluz71** [comment]
Can you test on another terminal such as Alacritty.

Also change to a different NERDFont, maybe update to the very latest NERDFont, or alternatively downgrade to an older font.

Try different combinations and see what happens. You are going to have to test since at the moment I can’t repeat your issue.

**Comment by bluz71** [comment]
Hello @qbnil,

I installed kitty (on Linux) and was unable to repeat this issue.

It definitely looks to be a file icon issue where the icon itself weirdly is taking up two characters, the file icon (Lua icon) and an erroneous extra space (both blue). I can not explain this.

Honestly speaking, I suspect you have an interaction action issue between your kitty configuration, nerd-font and Neovim web-devicons.

There is no way I can debug this remotely.

My recommendations to you:

- Update to the newest version of [nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons)
- Switch to a different and new NERDFont (temporarily)
- Experiment with a different terminal emulate such WezTerm or Alacritty (temporarily)
- Simplify your kitty setup with regard to fonts (do not do anything exotic such as line and character spacing)

I do not believe there is an underlying bug in this _linefly_ plugin otherwise I would have seen more bug reports like yours.

For now I am closing this down; if you provide a series of steps (including a minimal Neovim configuration file) that can reproduce the issue I will re-open.

Best of luck debugging since there is nothing I can do to fix this for you, you will have to get your hands dirty with some of the recommendations I have provided.

Cheers.
